app/controllers/routes_revealer/routes_controller.rb

```diff
@@ -31,6 +32,17 @@ def asset_route
       Rails.application.class.config.assets.prefix
     end
 
+    def public_folder
+      files = Dir.glob('public/**/*')
+      files.map! do |file|
+        file[6..-1]
```

Comment:
Interesting. Part of me wants you to dynamically generate the `6`, but really that's not necessary. Rather elegant really.